mempool/mempool.go

```diff
@@ -1,43 +1,67 @@
 package mempool
 
 import (
+	"crypto/sha256"
+	"errors"
 	"fmt"
+	"math"
 
 	abci "github.com/tendermint/tendermint/abci/types"
-	"github.com/tendermint/tendermint/p2p"
 	"github.com/tendermint/tendermint/types"
 )
 
+const (
+	MempoolChannel = byte(0x30)
```

Comment:
Isn't this an implementation detail of the tendermint p2p layer?

Reply:
Good catch!